lib/ui/views/login_view.dart

```diff
@@ -123,10 +142,17 @@ class _LoginViewState extends State<LoginView> {
                                       (Set<MaterialState> states) {
                                         if (states
                                             .contains(MaterialState.disabled)) {
-                                          return Colors.white38;
+                                          return getButtonColor(
+                                              model.getAppTheme(),
+                                              context,
+                                              Colors.white38,
+                                              Colors.grey[800]);
```

Comment:
With the example, I showed you this section should be revert as:
```suggestion
                                          return Colors.white38;
```